passage/src/main/java/id/passage/android/Passage.kt

```diff
@@ -1,759 +1,66 @@
-import id.passage.android.model.MagicLink
-import id.passage.android.model.RegisterMagicLinkRequest
-import id.passage.android.model.RegisterOneTimePasscodeRequest
-import id.passage.android.model.RegisterWebAuthnFinishRequest
-import id.passage.android.model.RegisterWebAuthnStartRequest
-import id.passage.client.infrastructure.ApiClient
 import okhttp3.OkHttpClient
 
 @Suppress("unused", "RedundantVisibilityModifier", "RedundantModalityModifier")
 public final class Passage(
-    private val activity: Activity,
+    activity: Activity,
     appId: String? = null,
 ) {
+    // region Private VARIABLES
+    var passageApp: PassageApp
+    var passagePasskey: PassagePasskey
+    var passageMagicLink: PassageMagicLink
+    var passageHosted: PassageHosted
+    var passageOneTimePasscode: PassageOneTimePasscode
+    var passageSocial: PassageSocial
+    var passageCurrentUser: PassageCurrentUser
+    var tokenStore: PassageTokenStore
+    var passageClient: OkHttpClient
+
     // region CONSTANTS AND SINGLETON VARIABLES
     internal companion object {
         internal const val TAG = "Passage"
         internal var BASE_PATH = "https://auth.passage.id/v1"
         internal lateinit var appId: String
         internal lateinit var authOrigin: String
         internal var language: String? = null
-
-        public fun setAuthToken(token: String?) {
-            ApiClient.accessToken = token
-        }
-
-        public fun getAuthToken(): String? {
-            return ApiClient.accessToken
-        }
     }
-    // endregion
-
-    // region INSTANCE VARIABLES
-
-    public lateinit var tokenStore: PassageTokenStore
-
-    private var isUsingTokenStore = false
-
-    private val passageClient: OkHttpClient by lazy {
-        val userAgent = WebSettings.getDefaultUserAgent(activity) ?: "Android"
-        OkHttpClient.Builder()
-            .addNetworkInterceptor { chain ->
-                chain.proceed(
-                    chain.request()
-                        .newBuilder()
-                        .header("User-Agent", userAgent)
-                        .build(),
-                )
-            }
-            .build()
-    }
-
-    // endregion
 
     // region INITIALIZATION
 
     init {
         authOrigin = getRequiredResourceFromApp(activity, "passage_auth_origin")
+        val userAgent = WebSettings.getDefaultUserAgent(activity) ?: "Android"
+        passageClient =
+            OkHttpClient
+                .Builder()
+                .addNetworkInterceptor { chain ->
+                    chain.proceed(
+                        chain
+                            .request()
+                            .newBuilder()
+                            .header("User-Agent", userAgent)
+                            .build(),
+                    )
+                }.build()
         Companion.appId = appId ?: getRequiredResourceFromApp(activity, "passage_app_id")
         language = getOptionalResourceFromApp(activity, "passage_language")
```

Comment:
These should be removed. We no longer want these values to be set from the `strings.xml` file - only `passage_auth_origin` should be set there (since it is a required value in `AndroidManifest.xml`.

### app id
App id should be part of the class constructor.

### language
Language strings should be passed as arguments in the methods that require it (OTP and ML login/register, and user information update methods)
```suggestion
```